zyte_common_items/pages.py

```diff
@@ -54,6 +61,36 @@ def _get_metadata_class(cls: type) -> Optional[Type[MetadataT]]:
     return get_generic_param(cls, HasMetadata)
 
 
+class PriceMixin(FieldsMixin):
+    """Provides price-related field implementations."""
+
+    _parsed_price: Optional[Price] = None
+
+    async def _get_parsed_price(self) -> Optional[Price]:
+        if self._parsed_price is None:
+            price_field = getattr(self, "price", object())
```

Comment:
Tests also pass like this:

```suggestion
            price_field = getattr(self, "price", None)
```

Reply:
Oh, it looks like I've added two fixes for the same problem.